### PR TITLE
Add v2 node API doc

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: api/rest/v2/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+[REST](https://en.wikipedia.org/wiki/Representational_state_transfer) library
+that enables developers to programmatically access data in the Data Commons
+knowledge graph. This package allows users to explore the structure of the
+graph, integrate statistics from the graph into data analysis applications and
+much more.
+
+## Getting Started
+
+First time using the Data Commons API, or just need a refresher? Take a look at
+our [Getting Started Guide](/api/rest/v1/getting_started).
```
Comment (beets) [inline api/rest/v2/index.md:23]: could you add a todo here to point to a new version of this? just so we don't miss it.

File: api/rest/v2/node.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+Fetches node information for edges and neighboring nodes. This is useful for
+finding local connections between nodes of the Data Commons knowledge graph.
+More specifically, this API can perform the following tasks:
+
+- Get all property labels associated with individual or multiple nodes.
+- Get the values of a property for individual or multiple nodes. These can also
+  be chained for multiple degrees in the graph.
+- Get all connected nodes that are linked with invidiual or mutiple nodes.
+
+Data Commons represents node relation as directed edges between nodes, or
```
Comment (beets) [inline api/rest/v2/node.md:22]:
```suggestion
Data Commons represents node relations as directed edges between nodes, or
```

**Review by beets — APPROVED** [review]
this is nice - thanks for getting this set up!

File: api/rest/v2/index.md (hunk 1 above)
Comment (shifucun) [reply api/rest/v2/index.md:24]: Done

File: api/rest/v2/node.md (hunk 2 above)
Comment (shifucun) [reply api/rest/v2/node.md]: Done

**Review by shifucun — COMMENTED** [review]
thanks!

**Pull request merged**